src/app-core.js

```diff
@@ -1,7 +1,9 @@
 import * as HAWS from 'home-assistant-js-websocket';
+import getPolyglot from './translation';
```

Comment:
We should not have the translations be part of the `app-core.js`. Instead, they should be part of the HTML bundle. We don't have to load the translations to start connecting to the server. `app-core.js` has to stay super lean.